src/store/rolesStore.js

```diff
@@ -0,0 +1,8 @@
+import create from 'zustand';
+
+const useRolesStore = create((set) => ({
+  roles: {},
```

Comment:
I think `roles` are for "users" so I think we can have them under the userStore and not create a separate one?

Reply:
resolved in [52ebd6b](https://github.com/RocketChat/EmbeddedChat/pull/115/commits/52ebd6bd133397d706cfb7139afc8bac43070fb5)